docs/Scenario_editor.md

```diff
@@ -13,15 +25,15 @@
     *For more information, check [here](https://docs.unrealengine.com/en-US/Programming/Development/VisualStudioSetup/index.html)*
 
 
-2. Download the monoDrive Simulator Editor from [here](https://www.monodrive.io/register).
+4. Download the monoDrive Simulator Editor from [here](https://www.monodrive.io/register).
 
-3. Open VehicleAI_Editor zip file and extract all files
+5. Open VehicleAI_Editor zip file and extract all files
 
     <div class="img_container">
     <img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/master/docs/LV_client/quick_start_img/sensor_editor_extract.png"/>
     </div>
 
-3. Go to your VehicleAI_Editor directory and find VehicleAI.uproject
+6. Go to your VehicleAI_Editor directory and find VehicleAI.uproject
```

Comment:
Numbering isn't actually important in markdown (the render will properly number the list no matter what number you have here). 

I usually just create a list of ones to make re-ordering easier (not that you have to do that here, just FYI):

```markdown
1. Item one
1. Item two
1. Item three
``` 

Renders as:

1. Item one
1. Item two
1. Item three